ipasetup.py.in

```diff
@@ -19,20 +19,21 @@ import sys
 
 
 PACKAGE_VERSION = {
-    'cryptography': 'cryptography >= 0.9',
+    'cryptography': 'cryptography >= 1.5',
     'dnspython': 'dnspython >= 1.11.1',
-    'gssapi': 'gssapi > 1.1.2',
+    'gssapi': 'gssapi >= 1.2.0',
```

Comment:
AFAIK we had a bug somewhere that 0.9 and older versions break Custodia actually.